src/utils/api/getBlockchainTransactions.ts

```diff
@@ -3,12 +3,29 @@
 //
 
 import { IGetBlockchainTransactionResult } from "types/NXTAPI";
-import { API } from "./api";
+import { API, IAPICall } from "./api";
+import { BASEURL } from "./constants";
+
+interface IGetBlockchainTransactionsParams extends IAPICall {
+  params: {
+    account: string;
```

Comment:
this object structure seems to be repeated a lot. perhaps consider a type that defines an account (only) is required for the call, so it doesn't have to be repeated.

do not do my suggestion here if you expect more params need to be added and for now you're just doing the basic call in all of these api areas.

Reply:
I could see this particular call eventually having additional (optional) params but I think your suggestion is legitimate for other calls. 

I won't take any action for now but as more calls are fleshed out it could be good to bring this to my attention again.